client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -51,28 +61,74 @@ const DesignSelector: FunctionComponent = () => {
 			</div>
 			<div className="design-selector__design-grid">
 				<div className="design-selector__grid">
-					{ designs.featured.map( design => (
-						<button
-							key={ design.slug }
-							className={ classnames(
-								'design-selector__design-option',
-								design.slug === selectedDesign?.slug ? 'selected' : null
-							) }
-							onClick={ () => {
-								setSelectedDesign( design );
-								if ( isEnabled( 'gutenboarding/style-preview' ) ) {
-									push( makePath( Step.Style ) );
-								}
-							} }
-						>
-							<div className="design-selector__image-frame">
-								<img alt={ design.title } src={ design.src } srcSet={ design.srcset } />
-							</div>
-							<span className="design-selector__option-overlay">
-								<span className="design-selector__option-name">{ design.title }</span>
-							</span>
-						</button>
-					) ) }
+					{ designs.featured.map( design => {
+						const isFocused = hoverDesigns.has( design.slug ) || focusDesigns.has( design.slug );
+						return (
+							<Spring
+								native
+								key={ design.slug }
+								from={ ZOOM_OFF }
+								to={ isFocused ? ZOOM_ON : ZOOM_OFF }
+							>
+								{ ( props: React.CSSProperties ) => (
+									<animated.button
+										style={ props }
+										onMouseEnter={ () =>
+											setHoverDesigns( s => {
+												const nextState = new Set( s );
+												nextState.add( design.slug );
+												return nextState;
+											} )
```

Comment:
Could do 
```suggestion
												setHoverDesigns( s => new Set( [ ...s, design.slug ] ) )
```

I think?